R/visualize.R

```diff
@@ -19,7 +19,7 @@
 #' @return A ggplot of MetAnnotate data
 #' @export
 generate_ggplot <- function(metannotate_data, hit_totals, plotting_colour_data,
-                            plotting_taxon, normalizing_HMM, plot_type = "bar",
+                            plotting_taxon, normalizing_HMM, plot_type = "bar", wrap = "vertical",
```

Comment:
@NickiShaw Can you also add a description of `wrap` as a `@param` entry in the function header above? This will be turned into help documentation for the function inside of R.